### ISSUE TITLE
XML with entities like &lt; get screwed up.

### ISSUE THREAD

**Issue by forbesmyester** [TomWright/dasel#61]
**Describe the bug**
Escaping in XML does not work properly (when the string is already there)

**To Reproduce**
Save file as es_systems.cfg

```xml
<?xml version="1.0"?>
<systemList>
  <system>
    <name>retropie</name>
    <fullname>RetroPie</fullname>
    <path>/home/fozz/RetroPie/retropiemenu</path>
    <extension>.rp .sh</extension>
    <command>sudo /home/fozz/RetroPie-Setup/retropie_packages.sh retropiemenu launch %ROM% &lt;/dev/tty &gt;/dev/tty</command>
    <platform/>
    <theme>retropie</theme>
  </system>
</systemList>
```

and run the command

```shell
cat es_systems.cfg | dasel put string -p xml a .systemList x
```

**Expected behavior**
The `&lt;/dev/tty &gt;` should have been retained, it also should have not generated an invalid document (it put in `<` instead of `&lt;`)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Version: development - 971ac931d97a2bacaf8103633424d5058663d592

**Comment by TomWright** [comment]
Thanks for raising this issue @forbesmyester.

This is an issue with the XML parser being used - I've raised an issue with them: https://github.com/clbanning/mxj/issues/84

**Comment by TomWright** [comment]
This has been fixed as of `v1.9.1`.

```
$ echo '<systemList>
  <system>
    <name>retropie</name>
    <fullname>RetroPie</fullname>
    <path>/home/fozz/RetroPie/retropiemenu</path>
    <extension>.rp .sh</extension>
    <command>sudo /home/fozz/RetroPie-Setup/retropie_packages.sh retropiemenu launch %ROM% &lt;/dev/tty &gt;/dev/tty</command>
    <platform/>
    <theme>retropie</theme>
  </system>
</systemList>' | dasel put string -p xml .newElement newValue
<doc>
  <newElement>newValue</newElement>
  <systemList>
    <system>
      <command>sudo /home/fozz/RetroPie-Setup/retropie_packages.sh retropiemenu launch %ROM% &lt;/dev/tty &gt;/dev/tty</command>
      <extension>.rp .sh</extension>
      <fullname>RetroPie</fullname>
      <name>retropie</name>
      <path>/home/fozz/RetroPie/retropiemenu</path>
      <platform/>
      <theme>retropie</theme>
    </system>
  </systemList>
</doc>
```

**Comment by forbesmyester** [comment]
Awesome, efficiency. Thanks :-)
